main/templates/keys.xml

```diff
@@ -30,4 +30,7 @@
     <!-- Geocaching.su -->
     <string name="su_consumer_key" translatable="false">@su.consumer.key@</string>
     <string name="su_consumer_secret" translatable="false">@su.consumer.secret@</string>
+
+    <!-- Geocaching.com public ALC API -->
+    <string name="alc_consumer_key" translatable="false">@alc.consumer.key@</string>
```

Comment:
As documented in https://github.com/cgeo/cgeo#api-keys-installation the key management has two parts - in the `main/template/keys.xml` you define the part for using the key in the code. And in `templates/private.properties` you define the point where the user should put his keys. 

The workflow described in the Readme works.

@baiti Please add the missing entry in `templates/private.properties` and maybe also a short text in the Readme.

Reply:
> As documented in https://github.com/cgeo/cgeo#api-keys-installation the key management has two parts - in the `main/template/keys.xml` you define the part for using the key in the code. And in `templates/private.properties` you define the point where the user should put his keys.
> 
> The workflow described in the Readme works.

Yes, it works, but it confused me. `private.properties` we take from `./templates/` and `keys.xml` we update in `./main/templates/`. 
> 
> @baiti Please add the missing entry in `templates/private.properties` and maybe also a short text in the Readme.

first part is done and commited. Not sure about what to write into the READ.me.